<?xml version="1.0" encoding="UTF-8"?>
<launch>
   
    <node name="tflite"	pkg="tf_ros_detection"      type="tflite"  	respawn="true" 	output="screen"><!-- launch-prefix="xterm -e gdb !args"-->
                <remap from="cam_in" 			to="/stereo_cam_node/image_raw"/>
		<param name="labels_dir"	type="string" 	value="/mscoco_label_map2.pbtxt" /> 
		<param name="models_dir"	type="string" 	value="/home/ubuntu/Downloads/coco_mobile/detect.tflite" />
		<!-- /home/ubuntu/Downloads/coco_mobile/detect.tflite , /home/ubuntu/Downloads/facenet_keras.tflite  , /home/ubuntu/Downloads/62faces_float.tflite -->
 		<param name="threshold_Score" value="0.7" />
 		<param name="threshold_IOU" value="0.8" />
     </node>

</launch>
